src/app/TradeApp/index.tsx

```diff
@@ -20,25 +21,37 @@ const Trading = React.lazy(
     ),
 )
 
-const Menu: React.FC = () => (
-  <Frame>
-    <footer>Menu</footer>
-  </Frame>
-)
-
 const NavTools: React.FC = () => (
   <Frame>
     <footer>Tools</footer>
   </Frame>
 )
 
-export const TradeApp: React.FC = () => (
-  <GenericLayout menu={<Menu />} navTools={<NavTools />}>
-    <React.Suspense fallback={null}>
-      <Switch>
-        <Route path="/v2" exact component={Trading} />
-        <Route component={NotFound} />
-      </Switch>
-    </React.Suspense>
-  </GenericLayout>
-)
+export const TradeApp: React.FC = () => {
+  const menu = (
+    <Menu>
+      <li>
+        <a href="#">Trade</a>
+      </li>
+      <li>
+        <a href="#">Swap</a>
+      </li>
+      <li>
+        <a href="#">Liquidity</a>
```

Comment:
Can we already use `Link`components, just to not change them later.

Reply:
I think it's an ordered list, where arguably order matters? OL is a valid semantic list. A <div> with <a>'s work, but arguably an ordered (or unordered) list is more correct in terms of accessibility?

Reply:
but in reality we don't even have pages... but as u please

Reply:
done